packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -15,7 +13,7 @@ import 'utils.dart';
 ///
 /// Setting this to null disables all integration with the browser history.
 void setUrlStrategy(UrlStrategy strategy) {
-  jsSetUrlStrategy(convertToJsUrlStrategy(strategy));
+  jsSetUrlStrategy(convertToJsUrlStrategy(strategy)!);
```

Comment:
Nulls are allowed it seems, so `!` is probably incorrect.

Reply:
Let's check for `strategy == null` here and avoid passing it through `convertToJsUrlStrategy` if it's null.

Reply:
This is done too, thanks for noticing the `!`, I totally misread this API!